### ISSUE TITLE
Suggestion: Enable multiple tags

### ISSUE THREAD

**Issue by oaklandgit** [replit/kaboom#386]
It would be useful to be able to target multiple tags at once by passing in an array of tags wherever tags can be used:
```
k.every(["enemy","bullet"], (obj) => {
  obj.paused = true
})
```
Or something like that.

**Comment by slmjkdbtl** [comment]
Implemented in #387 !

**Comment by oaklandgit** [comment]
Awesome! Thank you @slmjkdbtl
